encrypted-media-respec.html

```diff
@@ -1240,7 +1259,31 @@
                         </dd>
                         <dt><a def-id="persistent-usage-record-session"></a></dt>
                         <dd>
-                          <p>Let <var>requested license type</var> be a non-persistable license that will persist a <var>record of key usage</var>.</p>
+                          <ol>
+                            <li>
+                              <p>Initialize this object's <var>record of key usage</var> as follows.</p>
+                              <ul>
+                                <li>
+                                  <p>
+                                    Set the list of <a def-id="key-id">key IDs</a> <a href="#known-key">known</a> to the session to the empty list.
```

Comment:
nit: I'm not sure "the empty list" is a thing like "the empty string." Use "an"?